python/lsst/ctrl/pool/parallel.py

```diff
@@ -334,6 +336,10 @@ def makeBatch(self, args):
                       'submit': 'batchSubmit',
                       'options': 'batchOptions',
                       }
+
+        if not BATCH_TYPES[args.batchType]:
```

Comment:
Maybe it's not necessary.  PEP8 says:
> Comparisons to singletons like None should always be done with is or is not , never the equality operators.
But you're not using an equality operator.  I guess I was just expressing my personal preference, so feel free to ignore me.